crates/query-engine/metadata/src/metadata/native_queries.rs

```diff
@@ -75,42 +214,13 @@ impl Serialize for NativeQuerySql {
                 }
             }
         }
-        serializer.serialize_str(&sql)
-    }
-}
-
-struct NQVisitor;
```

Comment:
@plcplc taught me that we can avoid all of this cruft by just implementing From/Into.

Reply:
🌺